### ISSUE TITLE
Retain winds in AABW and NADW formation regions along with the equator

### ISSUE THREAD

**Issue by dhruvbhagtani** [dhruvbhagtani/sfc-perturbation-expts#13]
This experiment has the same motivation as (#12 ) in that it is aimed at preserving a net overturning circulation. However, having zero winds at the equator could result in a strongly reduced heat input at equilibrium. Therefore, we keep winds in the equatorial regions along with the polar regions. We can expect the overturning to be similar to the control run, and if that happens, we can compare the resulting changes in subtropical ocean gyres.

**Comment by navidcy** [comment]
@dhruvbhagtani "is also aligned" reads vague. Perhaps you are thinking this experiment as the "sister-experiment" of some other? But it's not clear what you are referring to -- probably #12, is this correct?

Similarly to #12: why this falls under the project `Model Improvements`?

**Comment by dhruvbhagtani** [comment]
Yes, you are right. I have edited my comment. Now it should be clearer than before.

**Comment by dhruvbhagtani** [comment]
Here is the case for winds present in:
1. Southern Hemisphere
2. NADW formation region (polewards of 50 degree north), and
3. Equator (10 degrees N/S).

https://github.com/dhruvbhagtani/varying-surface-forcing/blob/main/Stress_eq_except_NANP/Diagnostics.ipynb

**Comment by dhruvbhagtani** [comment]
In this case, the ocean is able to take more heat at the equator, and upon looking closely, a small subtropical cell forms in this case. The overturning circulation plot shows a small subtropical cell in the Into-Pacific sector, which is reasonable, considering the fact that we have winds till 10 degree north, after which they start reducing to zero.

<img width="1655" alt="Screen Shot 2021-10-17 at 10 29 52 am" src="https://user-images.githubusercontent.com/29700296/137612401-fc230517-2b65-4346-b382-8dfa8c34218f.png"
<img width="868" alt="Screen Shot 2021-10-17 at 10 30 58 am" src="https://user-images.githubusercontent.com/29700296/137612463-5eb0ef3c-4b8d-45b3-a28b-9ae78edf7aab.png">
>

**Comment by dhruvbhagtani** [comment]
Also, since we are talking about overturning, there is a more "complete" overturning than the case where winds are turned off in the equator as well.
<img width="1630" alt="Screen Shot 2021-10-17 at 10 30 43 am" src="https://user-images.githubusercontent.com/29700296/137612535-d4c79cb7-9a09-4b34-bcb2-396f7b8f73eb.png">

**Comment by dhruvbhagtani** [comment]
Another surprising result is that the Drake Passage transport has grown stronger by 8-10 Sv in the Nostress_0to50N and Nostress_10to50N cases. Something related to this is the Weddell gyre, which has decreased in strength, as shown in the screenshot. I think the two could be related, and I am reading [Patmore 2019](https://journals.ametsoc.org/view/journals/phoc/49/12/jpo-d-19-0083.1.xml) to gain more info about this.
<img width="1729" alt="Screen Shot 2021-10-17 at 10 31 12 am" src="https://user-images.githubusercontent.com/29700296/137612647-1bd74481-e198-433a-8fbd-8fd5d1dcaa35.png">
<img width="1757" alt="Screen Shot 2021-10-17 at 10 30 18 am" src="https://user-images.githubusercontent.com/29700296/137612637-37dd2c34-ed0a-4c76-a84b-c832acaf7805.png">

**Comment by AndyHoggANU** [comment]
For the Drake Passage transport, would be worth comparing Southern Ocean stratification?

For the overturning, the northern branch looks pretty similar to control here. Have we looked at the heat fluxes, gyre response, etc?
